src/components/Personalization/constants/loggerMessages.js

```diff
@@ -0,0 +1,5 @@
+export const decisionsDeprecatedWarning =
+  "The decisions property is decisionsDeprecatedWarning. Please use the propositions property instead.";
+export const authoringEnabled = "Rendering is disabled, authoring mode.";
+export const redirectExecutionError =
+  "An error occurred while executing the redirect offer.";
```

Comment:
For consistency, follow [these conventions](https://github.com/adobe/alloy/wiki/Conventions#enumerations), specifically using all caps for the constant names and changing the filename to `loggerMessage.js`